.taskcluster.yml

```diff
@@ -47,31 +106,39 @@ tasks:
             # "github-worker".
             workerType: wpt-docker-worker
             metadata:
-              name: wpt-${browser.name}-${browser.channel}-${chunk[0]}-${chunk[1]}
-              description: >-
-                A subset of WPT's "${chunk[0]}" tests (chunk number ${chunk[1]}
-                of ${chunk[2]}), run in the ${browser.channel} release of
-                ${browser.name}.
-              owner: ${event.pusher.email}
+              name: ${operation.name}
+              description: ${operation.description}
+              owner: ${event.pull_request.user.login}@users.noreply.github.com
```

Comment:
I meant to ask what the thinking was behind this change (not necessarily a problem; just wondering).

Reply:
git has incorrectly interpreted this section as a change to existing code. Actually, this is configuration for the new pull request tasks, so the "owner" specified is not a change but an addition (the "owner" for push-triggered tasks is unchanged). The two differ because in the context of a pull request, [the `event` object doesn't have a `pusher` property](https://developer.github.com/v3/activity/events/types/#pullrequestevent). TaskCluster requires a value here, so I cribbed this e-mail address from [an example in the TaskCluster docs](https://docs.taskcluster.net/docs/reference/integrations/taskcluster-github/docs/taskcluster-yml-v1#example).